WcaOnRails/app/models/round.rb

```diff
@@ -40,6 +40,40 @@ def event
     Event.c_find(competition_event.event_id)
   end
 
+  # Compute a round type id from round information
+  def round_type_id
```

Comment:
@SAuroux I had to move the code around a bit, the function computing the round type id is here.
Could you please take a quick look?
I believe I did this right, but I'd definitely be more comfortable with a confirmation from the WRT.

Reply:
@viroulep yes, the logic looks perfectly fine to me!

Reply:
Awesome thanks Sébastien!

I'll fix @jfly's nitpicks and merge this :)